src/main/java/uk/gov/justice/digital/oasys/service/AssessmentsService.java

```diff
@@ -72,10 +72,6 @@ private AssessmentDto populateAssessmentDto(Assessment assessment) {
     }
 
     private Boolean calculateChildSafeguardingIndicated(String assessmentType, Long oasysSetId) {
-        if (!LAYER_3.equalsIgnoreCase(assessmentType)) {
-            return null;
-        }
-
```

Comment:
What happens by removing this if it is a Layer3 asessment, is it just going to give a nonense score, or return null anyway on  line 77?

Reply:
The questions that identify if the flag should be set are in the ROSH section. I think it's possible for a layer 1 to have a ROSH. If not as you say it will just end up as Null.